dspace-api/src/test/java/org/dspace/eperson/GroupTest.java

```diff
@@ -206,6 +206,9 @@ public void findAllNameSort() throws SQLException {
         List<String> names = new ArrayList<>();
         List<String> sortedNames = new ArrayList<>();
         for (Group group : groups) {
+            if (group.getName() == null) {
+                continue;
```

Comment:
I also do wonder about this change. If groups are getting into the system with empty names, then our tests should be able to handle it. Does this change end up hiding a larger issue?